packages/cli/test/unit/commands/env.test.ts

```diff
@@ -184,23 +184,5 @@ describe('env', () => {
 
       await expect(pullPromise).resolves.toEqual(0);
     });
-
-    it('should show a custom message when it fails to read a file', async () => {
```

Comment:
I think we should still have a test for this case. The output should say that the `.env` file was updated, but it should not show a diff.

Reply:
I re-added the test, but I think it's hard to test for this. In every case where the env file was updated, it outputs "Updated <env name> file". In most cases, it then also outputs "No changes found", but in the case where it can't parse the file, it doesn't output that. So we want to test that something _doesn't_ output, which I'm not sure is possible right now.

Reply:
Ah, right, because our output mocking changed. This will at least make sure there's no error in this case.